add-ons/che/README.adoc

```diff
@@ -0,0 +1,26 @@
+= Eclipse Che Addon
+
+This addon create Eclipse Che templates and image streams on OpenShift. In short this helps in setting up Eclipse Che 
```

Comment:
Probably start with

```
This addon helps in setting up Eclipse Che on Minishift inline with ..... and creats Eclipse Che templates and image streams on OpenShift. 
```

Reply:
thanks, the README is just a placeholder am @sunix is till reviewing it and need to add one more stuff to template (WIP) once it done will polish the README

Reply:
I will suggest just ` This addon sets up Eclipse Che on Minishift` is enough. Not sure why we are saying it create Eclipse Che templates and image streams as we are adding the required templates and image-streams but not creating them.